evalml/model_understanding/graphs.py

```diff
@@ -484,19 +498,39 @@ def graph_partial_dependence(pipeline, X, feature, grid_resolution=100):
     _go = import_or_raise("plotly.graph_objects", error_msg="Cannot find dependency plotly.graph_objects")
     if jupyter_check():
         import_or_raise("ipywidgets", warning=True)
+    if isinstance(pipeline, evalml.pipelines.MulticlassClassificationPipeline) and class_label is not None:
+        if class_label not in pipeline.classes_:
+            msg = f"Class {class_label} is not one of the classes the pipeline was fit on: {', '.join(list(pipeline.classes_))}"
+            raise ValueError(msg)
 
     part_dep = partial_dependence(pipeline, X, feature=feature, grid_resolution=grid_resolution)
     feature_name = str(feature)
     title = f"Partial Dependence of '{feature_name}'"
     layout = _go.Layout(title={'text': title},
                         xaxis={'title': f'{feature_name}', 'range': _calculate_axis_range(part_dep['feature_values'])},
-                        yaxis={'title': 'Partial Dependence', 'range': _calculate_axis_range(part_dep['partial_dependence'])})
-    data = []
-    data.append(_go.Scatter(x=part_dep['feature_values'],
+                        yaxis={'title': 'Partial Dependence', 'range': _calculate_axis_range(part_dep['partial_dependence'])},
+                        showlegend=False)
+    if isinstance(pipeline, evalml.pipelines.MulticlassClassificationPipeline):
+        class_labels = [class_label] if class_label is not None else pipeline.classes_
+        _subplots = import_or_raise("plotly.subplots", error_msg="Cannot find dependency plotly.graph_objects")
+        fig = _subplots.make_subplots(rows=1, cols=len(class_labels), subplot_titles=class_labels,
+                                      shared_xaxes=True, shared_yaxes=True)
+        for i, label in enumerate(class_labels):
+            # Plotly trace indexing begins at 1 so we add 1 to i
+            fig.add_trace(_go.Scatter(x=part_dep.loc[part_dep.class_label == label, 'feature_values'],
+                                      y=part_dep.loc[part_dep.class_label == label, 'partial_dependence'],
+                                      line=dict(width=3)),
+                          row=1, col=i + 1)
```

Comment:
@freddyaboulton  Great! Also if the dataset has a larger number of labels, it might be difficult for the user to see all the partial dependency plots in one row.
Maybe:
`_subplots.make_subplots(rows=(len(class_labels)+1) // 2, cols=2 ...etc)`
and then in line 523:
`row=(i+2) // 2` and `col=(i%2) + 1`.
Not really a big deal either way

Reply:
Sounds good! I think the only thing missing was that we should only use two columns in the case where `class_label=None` or else there would be an empty second column in the plot.

Reply:
Excellent!